test/gtest/uct/test_flush.cc

```diff
@@ -444,6 +482,24 @@ UCS_TEST_SKIP_COND_P(uct_flush_test, put_bcopy_flush_ep_no_comp,
     }
 }
 
+UCS_TEST_SKIP_COND_P(uct_flush_test, put_bcopy_flush_ep_remote,
+                     !check_caps(UCT_IFACE_FLAG_PUT_BCOPY) ||
+                     !check_caps(UCT_IFACE_FLAG_GET_BCOPY)) {
+    m_flush_flags = UCT_FLUSH_FLAG_REMOTE;
+
+    check_skip_test_flush_remote();
+    test_flush_put_bcopy(&uct_flush_test::flush_ep_nb);
+}
+
+UCS_TEST_SKIP_COND_P(uct_flush_test, put_zcopy_flush_ep_remote,
+                     !check_caps(UCT_IFACE_FLAG_PUT_BCOPY) ||
+                     !check_caps(UCT_IFACE_FLAG_GET_BCOPY)) {
+    m_flush_flags = UCT_FLUSH_FLAG_REMOTE;
```

Comment:
can we pass this as parameter to `(this->*flush)` from check_skip_test_flush_remote

Reply:
> can we pass this as parameter to `(this->*flush)` from check_skip_test_flush_remote

ok, moved into `check_skip_test_flush_remote`